building/build-debs/homeworld-admin-tools/src/infra.py

```diff
@@ -5,7 +5,9 @@
 
 
 def infra_admit(server_principal: str) -> None:
-    token = access.call_keyreq("bootstrap-token", server_principal, collect=True)
+    config = configuration.get_config()
+    principal_hostname = config.get_hostname(server_principal)
+    token = access.call_keyreq("bootstrap-token", principal_hostname, collect=True)
```

Comment:
Good thinking. Coercing the name is definitely the correct solution to this problem.